### DISCUSSION TITLE
"Options error: lport and --nobind don't make sense when used together" OpenVPN and PFSense

### DISCUSSION THREAD

**Discussion by Mustbewordsornumbers** [RaspAP/raspap-webgui#1002] [General]
If anyone has the same error in the title using the config file from the OpenVPN package in PFSense in RaspAP the solution is to delete the `lport 0` line in the config file. As per below.

```
dev tun
persist-tun
persist-key
ncp-disable
cipher AES-256-CBC
auth SHA256
tls-client
client
resolv-retry infinite
remote your server here 1195 udp4
lport 0  << Delete this line and save
```
I haven't had a problem using this same config file in many other devices and it may well be that I have set an option wrong in OpenVPN within PFSense. I have no idea as I am not an expert in OpenVPN. 

Took me a few hours to work this out as I couldn't understand why the config file from the OpenVPN package in my Synology NAS worked first time whereas the PFSense one never connected.

Hopefully having the error in the title will help someone else and many thanks to the developer for their work.

**Comment by billz** [comment]
Edit: formatting
